molecule/default/prepare.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Prepare
+  hosts: molecule
+  gather_facts: false
+  become: yes
+  become_method: sudo
+  vars_files:
+    - vars/vars.yml
+
+  tasks:
+    - name: Create a non-root sudoer user
+      ansible.builtin.user:
+        name: "{{ tas_single_node_user }}"
+        shell: /bin/bash
+        create_home: yes
+
+    - name: Grant passwordless sudo to new user
```

Comment:
Would it be possible to not use passwordless? That's closer to a real world scenario.